kindedj/src/main/java/io/kindedj/Hk.java

```diff
@@ -16,5 +16,14 @@
 
 package io.kindedj;
 
-public interface HK<F, A> {
+/**
+ * Hk, or higher kind, represents a value that wraps another value. It would be equivalent to F&lt;A&gt; if Java supported higher kinds.
```

Comment:
rather than "*represents a value that wraps another value*", I would prefer:
"represents a parametrized type"
also: s/supported/had support for/.